ThunderBasics/TSCSingleRequestLocationManager.m

```diff
@@ -146,34 +154,26 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
         CLLocation *newLocation = locations[0];
         
         // Debug the reported location
-        if (kPCWebServiceLocationManagerDebug) {
-            NSLog(@"PCWebServiceLocationManager: New location: %@", newLocation);
-            NSLog(@"PCWebServiceLocationManager: Horizontal accuracy: %f", newLocation.horizontalAccuracy);
-            NSLog(@"PCWebServiceLocationManager: Vertical accuracy: %f", newLocation.verticalAccuracy);
-        }
-        
+		os_log_debug(location_manager_log, "New location: %@", newLocation);
+		os_log_debug(location_manager_log, "Horizontal accuracy: %f", newLocation.horizontalAccuracy);
+		os_log_debug(location_manager_log, "Vertical accuracy: %f", newLocation.verticalAccuracy);
+		
         // If accuracy greater than 100 meters, it's too inaccurate
         if(newLocation.horizontalAccuracy > 100 && newLocation.verticalAccuracy > 100){
-            if (kPCWebServiceLocationManagerDebug) {
-                NSLog(@"PCWebServiceLocationManager: Accuracy poor, aborting...");
-            }
+			os_log_debug(location_manager_log, "Accuracy poor, aborting...");
```

Comment:
Going to be nice to be able to see this kind of stuff day to day in the console I think. Should help us understand how location locking is working better 👍